File: app/src/main/java/cn/imgrocket/imgrocket/TestActivity.java
package cn.imgrocket.imgrocket;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class TestActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_test);
        final Button buttonSubmit = findViewById(R.id.test_button_submit);
        final TextView textViewView = findViewById(R.id.test_textview_view);
        final EditText editTextInput = findViewById(R.id.test_edit_input);
        buttonSubmit.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                Function.INSTANCE.post("https://api.huhaorui.com/test.php", editTextInput.getText().toString(), new Function.Callback() {
                    @Override
                    public void onResponse(String result) {
                        textViewView.setText(result);
                    }
                });

            }
        });
        //我希望，在textViewView上显示出返回的东西
    }
}

File: app/src/main/java/cn/imgrocket/imgrocket/TestActivity.kt
package cn.imgrocket.imgrocket

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import cn.imgrocket.imgrocket.databinding.ActivityTestBinding
import cn.imgrocket.imgrocket.tool.Function
import cn.imgrocket.imgrocket.tool.Function.post

class TestActivity : AppCompatActivity() {
    private lateinit var binding: ActivityTestBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityTestBinding.inflate(layoutInflater)
        setContentView(binding.root)
        val buttonSubmit = binding.testButtonSubmit
        val textViewView = binding.testTextviewView
        val editTextInput = binding.testEditInput
        buttonSubmit.setOnClickListener {
            post("https://api.huhaorui.com/test.php", editTextInput.text.toString(), object : Function.Callback {
                override fun onResponse(result: String?) {
                    textViewView.text = result
                }
            })
        }
        //我希望，在textViewView上显示出返回的东西
    }
}